net-misc/eventd/eventd-0.21.0.ebuild

```diff
@@ -0,0 +1,122 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit linux-info systemd
+
+DESCRIPTION="A small daemon to act on remote or local events"
+HOMEPAGE="https://www.eventd.org/"
+SRC_URI="https://www.eventd.org/download/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-3+ LGPL-3+ MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="debug fbcon +introspection ipv6 libcanberra libnotify +notification
+	purple sound speech systemd test upnp websocket +X zeroconf"
+
+REQUIRED_USE="
+	X? ( notification )
+	fbcon? ( notification )
+	notification? ( || ( X fbcon ) )
+"
+
+COMMON_DEPEND="
+	>=dev-libs/glib-2.40:2
+	sys-apps/util-linux
+	introspection? ( >=dev-libs/gobject-introspection-1.42 )
+	libcanberra? ( media-libs/libcanberra )
+	libnotify? ( x11-libs/gdk-pixbuf:2 )
+	notification? (
+		x11-libs/cairo
+		x11-libs/pango
+		x11-libs/gdk-pixbuf:2
+		X? (
+			x11-libs/cairo[xcb]
+			x11-libs/libxcb:=
+			x11-libs/xcb-util
+			x11-libs/xcb-util-wm
+		)
+	)
+	purple? ( net-im/pidgin )
+	sound? (
+		media-libs/libsndfile
+		media-sound/pulseaudio
+	)
+	speech? ( app-accessibility/speech-dispatcher )
+	systemd? ( sys-apps/systemd:= )
+	upnp? ( net-libs/gssdp:= )
+	websocket? ( >=net-libs/libsoup-2.50:2.4 )
+	zeroconf? ( net-dns/avahi[dbus] )
+"
+DEPEND="${COMMON_DEPEND}
+	app-text/docbook-xml-dtd:4.5
+	app-text/docbook-xsl-stylesheets
+	dev-libs/libxslt
+	virtual/pkgconfig
+	fbcon? ( virtual/os-headers )
+"
+RDEPEND="${COMMON_DEPEND}
+	net-libs/glib-networking[ssl]
+"
+
+pkg_setup() {
+	if use ipv6; then
+		if ! use test; then
+			CONFIG_CHECK="~IPV6"
+		else
+			CONFIG_CHECK="IPV6"
+		fi
+
+		linux-info_pkg_setup
+	fi
+}
+
+src_prepare() {
+	default_src_prepare
```

Comment:
Not sure if you didn't do this intentionally but you can just use plain `default` here (it will call the right phase).